text-dream/src/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
add how to run tests to readme

Reply:
Done, although I have not written tests yet.